blueprints/aws-alb-crossplane/gatewayclassblueprint-aws-alb-crossplane.yaml

```diff
@@ -337,6 +346,67 @@ spec:
                 tv2.dk/gw: {{ .Gateway.metadata.namespace }}-{{ .Gateway.metadata.name }}
             toPort: 15021
             type: ingress
+      hpa: |
+        {{ if or (get .Values.hpa "minReplicas") (get .Values.hpa "maxReplicas") }}
```

Comment:
Should we check for minReplicas here? Do we need hpa when maxReplica is not defined?

Reply:
Good point. I would say yes, we still need the HPA. This is because Istio creates the Deployment and we have no other way to set a fixed number of replicas.

Reply:
Ah, ok 👍